integration_test/tests/power_bi_tests.rs

```diff
@@ -14,6 +14,75 @@ mod integration {
     use std::ptr::null_mut;
     use std::slice;
 
+    const BUFFER_LENGTH: SmallInt = 300;
+
+    macro_rules! test_get_info {
+        ($conn_handle:expr, $info_type: expr, $info_value_buffer_length: expr, $info_value_type: expr) => {{
+            let conn_handle = $conn_handle;
+            let info_type = $info_type;
+            let info_value_buffer_length = $info_value_buffer_length;
+            let info_value_type = $info_value_type;
+
+            let output_buffer = &mut [0u16; (BUFFER_LENGTH as usize - 1)] as *mut _;
+            let mut buffer = OutputBuffer {
+                output_buffer: output_buffer as Pointer,
+                data_length: *&mut 0,
+            };
+
+            let outcome = SQLGetInfoW(
+                conn_handle as HDbc,
+                info_type,
+                buffer.output_buffer,
+                info_value_buffer_length,
+                &mut buffer.data_length as &mut _,
+            );
+            assert_eq!(
+                SqlReturn::SUCCESS,
+                outcome,
+                "Expected {}, got {}. Diagnostic message is: {}",
+                sql_return_to_string(SqlReturn::SUCCESS),
+                sql_return_to_string(outcome),
+                get_sql_diagnostics(HandleType::Dbc, conn_handle as Handle)
+            );
+
+            let length = buffer.data_length.clone();
+            println!(
+                "{info_type:?} = {}\nLength is {length}",
+                match info_value_type {
+                    DataType::WChar => Into::<String>::into(buffer),
+                    DataType::USmallInt => Into::<u16>::into(buffer).to_string(),
+                }
+            );
+        }};
+    }
+
+    pub enum DataType {
```

Comment:
When I tried to add the `mongo-odbc-driver` to the dependencies for the `integration_test` crate, I get errors like the following:
 odbc32.lib(ODBC32.dll) : error LNK2005: SQLNumResultCols already defined in libmongoodbc.rlib(mongoodbc.p7zbupwc4osljyd.rcgu.o)

Which I suspect is from having odbc-sys as a dependency as well since the ODBC Sql* functions with the same names are defined there.  So it seemed that adding the enum here was the way to go.  I can look to see if there are alternatives if you'd like.

Reply:
oh, yes, that's right. It's Ok then. Maybe if we get rid of odbc-sys in the future, then we can revisit this.